doc/UltiSnips.txt

```diff
@@ -350,6 +357,12 @@ over the global variable.
 
 |UltiSnips-adding-snippets| explains which files are parsed for a given filetype.
 
+If only one directory is specified in this variable and this directory is
+specified by absolute path, UltiSnips will not look for snippets in
+&runtimepath, which can lead to significant speedup. So, the common case is:
```

Comment:
mention that you will not see/use snippets shipped with other plugins then, only the ones in this directory.